Tests/ApodiniTests/ModifierTests/DescriptionModifierTests.swift

```diff
@@ -0,0 +1,74 @@
+//
+// Created by Lorena Schlesinger on 10.01.21.
+//
+
+import Foundation
+import XCTest
+@testable import Apodini
+
+
+final class DescriptionModifierTests: ApodiniTests {
+    struct TestHandler: Handler {
+        @Parameter
+        var name: String
+
+        func handle() -> String {
+            "Hello \(name)"
+        }
+    }
+
+    struct TestComponentDescription: Component {
+        @PathParameter
+        var name: String
+
+        var content: some Component {
+            Group("a", $name) {
+                TestHandler(name: $name)
+                    .description("Returns greeting with name parameter.")
+            }
+        }
+    }
+    
+    struct TestComponentWithoutDescription: Component {
+        @PathParameter
+        var name: String
+
+        var content: some Component {
+            Group("a", $name) {
+                TestHandler(name: $name)
+            }
+        }
+    }
+
+    func testEndpointDescription() {
+        let modelBuilder = SharedSemanticModelBuilder(app)
+        let visitor = SyntaxTreeVisitor(semanticModelBuilders: [modelBuilder])
+        let testComponent = TestComponentDescription()
+        Group {
+            testComponent.content
+        }.accept(visitor)
+        visitor.finishParsing()
+
+        // swiftlint:disable force_unwrapping
+        let treeNodeA: EndpointsTreeNode = modelBuilder.rootNode.children.first!.children.first!
```

Comment:
nit: Could you use `XCTUnwrap(...)` instead of force-unwrapping & disabling the SwiftLint rule?

Reply:
Of course, thanks for the reminder, I was not aware of this ;)! solved by 5b6a526